huggingface/pytorch/buildspec-trcomp-1-9.yml

```diff
@@ -0,0 +1,48 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+base_framework: &BASE_FRAMEWORK pytorch
+framework: &FRAMEWORK !join [ "huggingface_", *BASE_FRAMEWORK, "_trcomp"]
+version: &VERSION 1.9.0
```

Comment:
HF is currently at 1.9.1
https://github.com/aws/deep-learning-containers/blob/master/available_images.md